src/app/app.module.ts

```diff
@@ -13,7 +17,9 @@ import { environment } from '../environments/environment';
   imports: [
     BrowserModule,
     AppRoutingModule,
-    ServiceWorkerModule.register('ngsw-worker.js', { enabled: environment.production })
+    ServiceWorkerModule.register('ngsw-worker.js', { enabled: environment.production }),
+    BrowserAnimationsModule,
+    AppMaterialModule
```

Comment:
here is an example of something I didnt do .. adding a comma in the end will make sure that line 16 does not show up as deleted and line 20 does not show up as added. We should probably add a comma in line 22 and add this to the linting rules

Reply:
We can come back to this after https://trello.com/c/vv8N1y0B/28-add-rule-to-tslint-to-require-comma-at-the-end-of-an-array-or-json